rust/noosphere-sphere/src/has.rs

```diff
@@ -105,7 +97,10 @@ where
     /// new version [Cid] of the sphere is returned. This method must be invoked
```

Comment:
followup: I bet there's a handful of `[Cid]` usage in docs now that should be `Link<MemoIpld>`

Reply:
Tried to fix as many of these as I could find :+1: